src/components/NewMessage/NewMessage.vue

```diff
@@ -466,6 +481,12 @@ export default {
 				}
 			}
 
+			if (this.upload) {
+				this.$emit('sent', this.text)
+				this.$store.dispatch('setCurrentMessageInput', { token: this.token, text: '' })
```

Comment:
There should be a smooth scrolling to the bottom after upload is **completely** finished but don't know where to emit the event

Reply:
Maybe we could use a preview from temp message behind loading spinner, so height will remain. But should check, how will that work for failed uploads

Reply:
I researched a bit: this has nothing to do with this PR, so I'd raise a follow-up here (or there is probably an issue for this already)

Pushed a new commit to use local images (from blob) and reduce server load and list flickering